.github/workflows/push.yml

```diff
@@ -11,6 +17,22 @@ jobs:
       group: ${{ github.workflow }}-${{ github.ref }}
       cancel-in-progress: true
     steps:
+      - name: Free Disk Space (Ubuntu)
```

Comment:
This is because we are currently installing `texlive-full` and previous tests in FLT shows that "out of disk space" could happen. This steps takes ~ 3min in CI.

It could be accelerated, see https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Lean.204.20project.20blueprint/near/396040302 for details.